src/components/select/select.ts

```diff
@@ -252,6 +252,14 @@ export default class IgcSelectComponent extends EventEmitterMixin<
     super.firstUpdated();
     await this.updateComplete;
 
+    if (this.hasPrefixes || this.hasSuffixes) {
+      this.requestUpdate();
+    }
+
+    if (this.helperText) {
+      this.requestUpdate();
+    }
```

Comment:
```suggestion
    if (this.hasPrefixes || this.hasSuffixes || this.helperText) {
      this.requestUpdate();
    }
```

Reply:
Maybe attach a general `slotchange` handler to the slots which re-renders the component when there is slotted content?